src/applications/personalization/profile-2/components/connected-apps/ConnectedApps.jsx

```diff
@@ -44,22 +47,47 @@ export class ConnectedApps extends Component {
         {apps &&
           !allAppsDeleted && (
             <p className="va-introtext vads-u-font-size--md">
-              You’ve given these third-party apps or websites access to some of
-              your Veteran data, like health or service records. You can remove
-              their access at any time by disconnecting the app. Disconnected
-              apps can’t receive any new data from VA, but may still have access
-              to information that you’ve previously shared.
+              Your VA.gov profile is connected to the third-party (non-VA) apps
+              listed below. If you want to stop sharing information with an app,
+              you can disconnect it from your profile at any time.
             </p>
           )}
 
         {!apps ||
-          (allAppsDeleted && (
-            <p className="va-introtext vads-u-font-size--md">
-              You don’t currently have any third-party apps or websites
-              connected to your Veteran data, like health or service records.
-              When you do, this is where you can manage them.
-            </p>
-          ))}
+          (allAppsDeleted &&
```

Comment:
This is another example of something that would be nice to clean up. I can be hard to follow the logic here and would be a little easier if all this conditional stuff was done outside of the `render` method.